zenodo/modules/utils/cli.py

```diff
@@ -407,3 +410,32 @@ def update_openaire_communities(path):
         unresolved_communities, indent=4, separators=(', ', ': '))))
     click.secho('{0}'.format(json.dumps(mapping, indent=4,
                                         separators=(', ', ': '))), fg='blue')
+
+
+@utils.command('list_failed_openaire_records')
+@with_appcontext
+def list_failed_openaire_records():
+    """Lists records that failed to get indexed in OpenAIRE."""
+    failed_record_recids = []
+    for key in current_cache.cache._read_clients.scan_iter(
+        match=(current_cache.cache.key_prefix + 'openaire_direct_index*')):
+            recid = key.split('openaire_direct_index:')[1]
+            failed_record_recids.append(key)
+    click.echo(failed_record_recids)
```

Comment:
```suggestion
    click.echo('\n'.join(failed_record_recids))
```

So that we get a one-recid-per-line output.